source/frontend/src/features/mapSideBar/acquisition/common/GenerateForm/hooks/useGenerateH0443.ts

```diff
@@ -62,10 +55,57 @@ export const useGenerateH0443 = () => {
             : [],
         owner_names: owners.map<string>(x => getOwnerName(x)) || [],
         owner_contact: contactOwner !== undefined ? new Api_GenerateOwner(contactOwner) : null,
-        property_coordinator: new Api_GeneratePerson(coordinatorPerson),
-        property_agent: new Api_GeneratePerson(agentPerson),
+        property_coordinator: null,
+        property_agent: null,
       };
 
+      // Get the property coordinator by checking if it's a person or an org.
+      if (propertyCoordinator) {
+        if (propertyCoordinator.personId) {
+          const personConceptResponse = await getPersonConcept(propertyCoordinator?.personId);
+
+          h0443Data.property_coordinator = new Api_GeneratePerson(personConceptResponse.data);
+        } else if (propertyCoordinator.organizationId) {
+          const organizationConceptResponse = await getOrganizationConcept(
+            propertyCoordinator?.organizationId,
+          );
+
+          if (
```

Comment:
@eddherrera for this one, it seems like we don't set a property coordinator for an org unless there is a primary contact. Is that true?

Reply:
Correct: It was specified as an acceptance criteria.

> If for an applicable case primary contact is not provided leave relevant fields blank

Reply:
Yes, that seemed odd to me as well when reviewing this PR because it is a different pattern of what we usually do, but JIRA was very clear about what they want...
![image](https://github.com/bcgov/PSP/assets/24322224/fabc6c1f-99c0-4f94-824b-969e213efc53)

Perhaps adding a comment to avoid future confusion?